tests/countries/test_lebanon.py

```diff
@@ -0,0 +1,372 @@
+            "2017-04-15",
+        )
+        self.assertNoHolidayName(name, set(range(2010, 2025)) - {2010, 2011, 2014, 2017})
+
+    def test_catholic_easter_monday(self):
+        name = "اثنين الفصح عند الطوائف الكاثوليكية"
+        self.assertHolidayName(
+            name,
+            "1986-03-31",
+            "1987-04-20",
+            "1988-04-04",
+            "1989-03-27",
+            "1990-04-16",
+        )
+        self.assertHolidayName(name, range(1986, 1995))
+        self.assertNoHolidayName(name, range(1978, 1986), range(1995, 2050))
+
+    def test_orthodox_easter_monday(self):
+        name = "اثنين الفصح عند الطوائف الأرثوذكسية"
+        self.assertHolidayName(
+            name,
+            "1986-05-05",
+            "1987-04-20",
+            "1988-04-11",
+            "1989-05-01",
+            "1990-04-16",
+        )
+        self.assertHolidayName(name, range(1986, 1995))
+        self.assertNoHolidayName(name, range(1978, 1986), range(1995, 2050))
+
+    def test_orthodox_easter_tuesday(self):
+        name = "ثلاثاء الفصح للطوائف الأرثوذكسية"
+        self.assertHolidayName(
+            name,
+            "1987-04-21",
+            "1990-04-17",
+        )
+        self.assertNoHolidayName(name, set(range(1978, 2050)) - {1987, 1990})
+
+    def test_labor_day(self):
+        name = "عيد العمل"
+        self.assertHolidayName(name, (f"{year}-05-01" for year in range(1978, 2050)))
+        dt = (
+            "2005-05-02",
+            "2011-05-02",
+            "2016-05-02",
+            "2022-05-02",
+        )
+        self.assertHolidayName(f"{name} (يُحتفل به)", dt)
+
+    def test_martyrs_day(self):
+        name = "عيد الشهداء"
+        self.assertHolidayName(
+            name,
+            "1978-05-07",
+            "1979-05-06",
+            "1992-05-10",
+            "1993-05-09",
+            "2006-05-07",
+            "2007-05-06",
+            "2008-05-04",
+            "2009-05-03",
+        )
+        self.assertHolidayName(name, (*range(1978, 1994), *range(2006, 2050)))
+        self.assertNoHolidayName(name, range(1994, 2006))
+
+    def test_resistance_and_liberation_day(self):
+        name = "عيد المقاومة والتحرير"
+        self.assertHolidayName(
+            name,
+            "2021-05-09",
+            "2022-05-08",
+            "2023-05-14",
+            "2024-05-12",
+            "2025-05-11",
+        )
+        self.assertHolidayName(name, range(2006, 2050))
+        self.assertNoHolidayName(name, range(1978, 2006))
+
+    def test_anniversary_of_the_tragedy_of_beirut_port_explosion(self):
+        name = "ذكرى مأساة انفجار مرفأ بيروت"
+        self.assertNoHolidayName(name)
+        self.assertHolidayName(
+            name, self.government_holidays, (f"{year}-08-04" for year in range(2021, 2050))
+        )
+        self.assertNoHolidayName(name, self.bank_holidays, range(1978, 2050))
+        self.assertNoHolidayName(name, self.government_holidays, range(1978, 2019))
```

Comment:
```suggestion
        self.assertNoHolidayName(name, self.government_holidays, range(1978, 2021))
        self.assertNoHolidayName(name, self.bank_holidays)
```
It seems to be the last fix.